edx/cc2olx
# [BD-27] [TNL-7948] Add Support for Canvas Internal Links

Courses from Canvas contain links using the key $CANVAS_OBJECT_REFERENCE$ . This is meant to represent an internal link to another location in the course. These links contain Common Cartridge identifiers, and are an allegory to edX's jump_to_id links. This PR fixes linking such that these links are properly dereferenced.

**JIRA tickets**: TNL-7948

**Dependencies**:  https://github.com/edx/cc2olx/pull/48

**Merge deadline**: None

**Testing instructions**:

1. Import a Canvas course into Studio via LMS, go to any page that has a broken link and verify that the link is broken
2. Checkout this PR, convert the course again and import it again
3. Verify that the broken link is now fixed

**Reviewers**
